bson/src/main/org/bson/codecs/jsr310/LocalTimeCodec.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ * Copyright 2018 Cezary Bartosiak
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.codecs.jsr310;
+
+import org.bson.BsonReader;
+import org.bson.BsonWriter;
+import org.bson.codecs.DecoderContext;
+import org.bson.codecs.EncoderContext;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+
+/**
+ * LocalTime Codec.
+ *
+ * <p>
+ * Encodes and decodes {@code LocalTime} objects to and from {@code DateTime}. Data is stored to millisecond accuracy.
```

Comment:
So this is consistent with how other ODMs represent LocalTime in MongoDB?  It still seems strange to me that this would be a DateTime rather than just an Int64.

Reply:
After discussions here and in the second PR I have realized all codecs should be configurable. Look at this issue: https://github.com/cbartosiak/bson-codecs-jsr310/issues/1 I think these few codecs you would like to be a part of the driver might follow this idea. For example I prefer `LocalTime` to be stored as `DateTime`, but I understand that you prefer `Int64`. Users should be able to decide what they want. What do you think?

Reply:
Spring and Morphia both default to date.  Spring defaults to todays date rather than Epoch.  So it follows prior art and makes a sensible default. 

If users need / want an alternative localTime value then they would have to register an alternative Codec.